README.md

```diff
@@ -109,11 +78,70 @@ s3://mybucket/aws-instance-scheduler/v1.4.0/instance-scheduler.template
 s3://mybucket/aws-instance-scheduler/v1.4.0/instance-scheduler-remote.template
 ```
 
-## Deploy
+### Deploy
 
 See the [AWS Instance Scheduler Implementation Guide](https://s3.amazonaws.com/solutions-reference/aws-instance-scheduler/latest/instance-scheduler.pdf) for deployment instructions, using the link to the instance-scheduler.template from your bucket, rather than the one for AWS Solutions. Ex. https://mybucket.s3.amazonaws.com/aws-instance-scheduler/v1.4.0.mybuild/instance-scheduler.template
 
-## CDK Documentation
+
+# Testing the Solution
+
+## Running Tests Locally
+to test the solution you will need to install tox
+
+```pip install tox```
+
+Then from the root directory of the solution
+```
+//run all unit tests
+tox
+
+//test just the lambda code
+tox -e lambda
+
+//test just the cdk code
+tox -e cdk
+```
+
+## Automated Testing Pipeline
+AWS Instance Scheduler comes with an automated testing pipeline that can be used to automatically test any changes you
```

Comment:
Wrong solution name. Also, for clarity, the solution does not "come with it". We don't want customers to think they are getting this automatically if they deploy from the templates. It's an optional method to validate the solution (and changes to it) in your own account. Under what circumstances might someone want to run it? Developing features, validating the solution in their own environment etc.

Reply:
fixed